src/migrations/043_userinfo_oidc.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from 'knex';
```

Comment:
This migration filename still needs a more 'recent' number. Check out the patch directory, most of the later ones all use a timestamp instead of a 3 digit number.

Reply:
oh hey I missed this! yeah I made a commit while waiting for something at work